notebooks/PBMC-ATAC-coverage_plots_and_motifs.ipynb

```diff
@@ -0,0 +1,1343 @@
+     "shell.execute_reply.started": "2025-03-19T00:17:41.252419Z"
+    },
+    "new_sheet": false
+   },
+   "outputs": [],
+   "source": [
+    "# Cell names should be consistent with fragments file\n",
+    "ad.obs['FragSample'] = ad.obs_names.str.split('#').str.get(1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3936b41d-5655-4ff1-bfa2-ae2cbc8ecc35",
+   "metadata": {
+    "button": false,
+    "new_sheet": false
+   },
+   "source": [
+    "## Barcode groups"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "8b87d45b-ea44-4c89-948a-5bdbcbb5ad30",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2021-01-11T04:25:18.933219Z",
+     "start_time": "2021-01-11T04:25:18.895463Z"
+    },
+    "button": false,
+    "execution": {
+     "iopub.execute_input": "2025-03-19T00:17:41.290062Z",
+     "iopub.status.busy": "2025-03-19T00:17:41.289684Z",
+     "iopub.status.idle": "2025-03-19T00:17:43.961030Z",
+     "shell.execute_reply": "2025-03-19T00:17:43.959181Z",
+     "shell.execute_reply.started": "2025-03-19T00:17:41.290042Z"
+    },
+    "new_sheet": false
+   },
+   "outputs": [
+    {
+     "ename": "KeyError",
+     "evalue": "'phenograph'",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mKeyError\u001b[0m                                  Traceback (most recent call last)",
+      "File \u001b[0;32m~/.conda/envs/singlecell/lib/python3.8/site-packages/pandas/core/indexes/base.py:3629\u001b[0m, in \u001b[0;36mIndex.get_loc\u001b[0;34m(self, key, method, tolerance)\u001b[0m\n\u001b[1;32m   3628\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m-> 3629\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_engine\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget_loc\u001b[49m\u001b[43m(\u001b[49m\u001b[43mcasted_key\u001b[49m\u001b[43m)\u001b[49m\n\u001b[1;32m   3630\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mKeyError\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m err:\n",
+      "File \u001b[0;32m~/.conda/envs/singlecell/lib/python3.8/site-packages/pandas/_libs/index.pyx:136\u001b[0m, in \u001b[0;36mpandas._libs.index.IndexEngine.get_loc\u001b[0;34m()\u001b[0m\n",
+      "File \u001b[0;32m~/.conda/envs/singlecell/lib/python3.8/site-packages/pandas/_libs/index.pyx:163\u001b[0m, in \u001b[0;36mpandas._libs.index.IndexEngine.get_loc\u001b[0;34m()\u001b[0m\n",
+      "File \u001b[0;32mpandas/_libs/hashtable_class_helper.pxi:5198\u001b[0m, in \u001b[0;36mpandas._libs.hashtable.PyObjectHashTable.get_item\u001b[0;34m()\u001b[0m\n",
+      "File \u001b[0;32mpandas/_libs/hashtable_class_helper.pxi:5206\u001b[0m, in \u001b[0;36mpandas._libs.hashtable.PyObjectHashTable.get_item\u001b[0;34m()\u001b[0m\n",
+      "\u001b[0;31mKeyError\u001b[0m: 'phenograph'",
```

Comment:
Looks like the notebook did not run through and failed here. Maybe needs a fix and rerun.